
from PyQt5 import QtCore, QtGui, QtWidgets
from .mplwidget import MplWidget

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.setStyleSheet("background-color: white;")
        MainWindow.setMinimumSize(QtCore.QSize(1574, 810))
        MainWindow.setMaximumSize(QtCore.QSize(1574, 810))
        MainWindow.resize(1574, 810)
        MainWindow.setToolButtonStyle(QtCore.Qt.ToolButtonFollowStyle)
        MainWindow.setTabShape(QtWidgets.QTabWidget.Rounded)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.gridLayoutWidget.setGeometry(QtCore.QRect(80, 150, 211, 51))
        self.gridLayoutWidget.setObjectName("gridLayoutWidget")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.gridLayoutWidget)
        self.gridLayout_3.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.Duracion_calibracion = QtWidgets.QLabel(self.gridLayoutWidget)
        self.Duracion_calibracion.setAlignment(QtCore.Qt.AlignCenter)
        self.Duracion_calibracion.setObjectName("Duracion_calibracion")
        self.gridLayout_3.addWidget(self.Duracion_calibracion, 0, 0, 1, 1)
        self.Label_duracion_calibracion = QtWidgets.QLabel(self.gridLayoutWidget)
        self.Label_duracion_calibracion.setText("")
        self.Label_duracion_calibracion.setObjectName("Label_duracion_calibracion")
        self.gridLayout_3.addWidget(self.Label_duracion_calibracion, 0, 1, 1, 1)
        self.Slider_calibracion = QtWidgets.QSlider(self.gridLayoutWidget)
        self.Slider_calibracion.setOrientation(QtCore.Qt.Horizontal)
        self.Slider_calibracion.setObjectName("Slider_calibracion")
        self.gridLayout_3.addWidget(self.Slider_calibracion, 1, 0, 1, 2)
        self.gridLayoutWidget_2 = QtWidgets.QWidget(self.centralwidget)
        self.gridLayoutWidget_2.setGeometry(QtCore.QRect(80, 270, 211, 51))
        self.gridLayoutWidget_2.setObjectName("gridLayoutWidget_2")
        self.gridLayout_4 = QtWidgets.QGridLayout(self.gridLayoutWidget_2)
        self.gridLayout_4.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_4.setObjectName("gridLayout_4")
        self.Slider_iniciar = QtWidgets.QSlider(self.gridLayoutWidget_2)
        self.Slider_iniciar.setOrientation(QtCore.Qt.Horizontal)
        self.Slider_iniciar.setObjectName("Slider_iniciar")
        self.gridLayout_4.addWidget(self.Slider_iniciar, 1, 0, 1, 2)
        self.Duracion_calibracion_2 = QtWidgets.QLabel(self.gridLayoutWidget_2)
        self.Duracion_calibracion_2.setAlignment(QtCore.Qt.AlignCenter)
        self.Duracion_calibracion_2.setObjectName("Duracion_calibracion_2")
        self.gridLayout_4.addWidget(self.Duracion_calibracion_2, 0, 0, 1, 1)
        self.Label_duracion_iniciar = QtWidgets.QLabel(self.gridLayoutWidget_2)
        self.Label_duracion_iniciar.setText("")
        self.Label_duracion_iniciar.setObjectName("Label_duracion_iniciar")
        self.gridLayout_4.addWidget(self.Label_duracion_iniciar, 0, 1, 1, 1)
        self.Figure = MplWidget(self.centralwidget)
        self.Figure.setGeometry(QtCore.QRect(350, 20, 1221, 711))
        self.Figure.setObjectName("Figure")
        self.Consola = QtWidgets.QTextBrowser(self.centralwidget)
        self.Consola.setGeometry(QtCore.QRect(30, 340, 311, 391))
        self.Consola.setObjectName("Consola")
        self.layoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget.setGeometry(QtCore.QRect(60, 110, 261, 30))
        self.layoutWidget.setObjectName("layoutWidget")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.layoutWidget)
        self.horizontalLayout_2.setContentsMargins(0, 0, 0, 0)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.Calibrar = QtWidgets.QPushButton(self.layoutWidget)
        self.Calibrar.setObjectName("Calibrar")
        self.horizontalLayout_2.addWidget(self.Calibrar)
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_2.addItem(spacerItem)
        self.Progreso_Calibrar = QtWidgets.QProgressBar(self.layoutWidget)
        self.Progreso_Calibrar.setEnabled(True)
        self.Progreso_Calibrar.setProperty("value", 0)
        self.Progreso_Calibrar.setTextVisible(True)
        self.Progreso_Calibrar.setInvertedAppearance(False)
        self.Progreso_Calibrar.setObjectName("Progreso_Calibrar")
        self.horizontalLayout_2.addWidget(self.Progreso_Calibrar)
        self.layoutWidget1 = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget1.setGeometry(QtCore.QRect(60, 230, 261, 30))
        self.layoutWidget1.setObjectName("layoutWidget1")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.layoutWidget1)
        self.horizontalLayout.setContentsMargins(0, 0, 0, 0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.Iniciar = QtWidgets.QPushButton(self.layoutWidget1)
        self.Iniciar.setObjectName("Iniciar")
        self.horizontalLayout.addWidget(self.Iniciar)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem1)
        self.Progreso_iniciar = QtWidgets.QProgressBar(self.layoutWidget1)
        self.Progreso_iniciar.setEnabled(True)
        self.Progreso_iniciar.setProperty("value", 0)
        self.Progreso_iniciar.setTextVisible(True)
        self.Progreso_iniciar.setInvertedAppearance(False)
        self.Progreso_iniciar.setObjectName("Progreso_iniciar")
        self.horizontalLayout.addWidget(self.Progreso_iniciar)
        self.gridLayoutWidget_5 = QtWidgets.QWidget(self.centralwidget)
        self.gridLayoutWidget_5.setGeometry(QtCore.QRect(40, 30, 291, 61))
        self.gridLayoutWidget_5.setObjectName("gridLayoutWidget_5")
        self.gridLayout_5 = QtWidgets.QGridLayout(self.gridLayoutWidget_5)
        self.gridLayout_5.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_5.setObjectName("gridLayout_5")
        self.horizontalLayout_5 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_5.setObjectName("horizontalLayout_5")
        self.Nombre_plot = QtWidgets.QLineEdit(self.gridLayoutWidget_5)
        self.Nombre_plot.setObjectName("Nombre_plot")
        self.horizontalLayout_5.addWidget(self.Nombre_plot)
        self.Agregar_plot = QtWidgets.QPushButton(self.gridLayoutWidget_5)
        self.Agregar_plot.setObjectName("Agregar_plot")
        self.horizontalLayout_5.addWidget(self.Agregar_plot)
        self.gridLayout_5.addLayout(self.horizontalLayout_5, 3, 0, 1, 1)
        self.label_7 = QtWidgets.QLabel(self.gridLayoutWidget_5)
        self.label_7.setTextFormat(QtCore.Qt.AutoText)
        self.label_7.setAlignment(QtCore.Qt.AlignCenter)
        self.label_7.setObjectName("label_7")
        self.gridLayout_5.addWidget(self.label_7, 2, 0, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1574, 26))
        self.menubar.setObjectName("menubar")
        self.menuOpciones = QtWidgets.QMenu(self.menubar)
        self.menuOpciones.setObjectName("menuOpciones")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionAbrir = QtWidgets.QAction(MainWindow)
        self.actionAbrir.setObjectName("actionAbrir")
        self.actionGuardar = QtWidgets.QAction(MainWindow)
        self.actionGuardar.setObjectName("actionGuardar")
        self.actionGuardar_como = QtWidgets.QAction(MainWindow)
        self.actionGuardar_como.setObjectName("actionGuardar_como")
        self.menuOpciones.addAction(self.actionAbrir)
        self.menuOpciones.addAction(self.actionGuardar)
        self.menuOpciones.addAction(self.actionGuardar_como)
        self.menubar.addAction(self.menuOpciones.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.Duracion_calibracion.setText(_translate("MainWindow", "Duración [s]"))
        self.Duracion_calibracion_2.setText(_translate("MainWindow", "Duración [s]"))
        self.Consola.setHtml(_translate("MainWindow", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
"<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
"p, li { white-space: pre-wrap; }\n"
"</style></head><body style=\" font-family:\'MS Shell Dlg 2\'; font-size:7.8pt; font-weight:400; font-style:normal;\">\n"
"<p align=\"center\" style=\"-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:\'MS Shell Dlg 2\';\"><br /></p></body></html>"))
        self.Calibrar.setText(_translate("MainWindow", "Calibrar"))
        self.Iniciar.setText(_translate("MainWindow", "Iniciar"))
        self.Agregar_plot.setText(_translate("MainWindow", "Agregar"))
        self.label_7.setText(_translate("MainWindow", "Agregar medición"))
        self.menuOpciones.setTitle(_translate("MainWindow", "Opciones"))
        self.actionAbrir.setText(_translate("MainWindow", "Abrir"))
        self.actionGuardar.setText(_translate("MainWindow", "Guardar"))
        self.actionGuardar_como.setText(_translate("MainWindow", "Guardar como.."))
        

